docs/src/lisp/overview.md

```diff
@@ -0,0 +1,20 @@
+# Lisp
+
+
+"Lisp is outdated, Lisp is slow, Lisp has too many parentheses" &mdash; what excuses do people come
+up with to once again resort to using clumsy mainstream scripting languages? While Lisp
+dates back to the 1960s, its foundational ideas are so simple and general that they transcend the
+concept of age, much like mathematics itself.
+
+Parentheses? It might come as a surprise, but the number of parentheses in Lisp is comparable to
+the number of parentheses (as well as brackets and braces) in other languages that lack significant
+indentation. However, in Lisp, we gain a beautiful property: while editing and navigating code,
+a programmer can operate on a syntax tree rather than individual characters and words. In fact,
+in Lisp, every expression can be traversed with proper editor support &mdash; moving forward, backward,
+to the parent, and to nested expressions, opposed to characters, words or in the best case type
+definitions and functions in other languages. This level of structural clarity and manipulation is a
+testament to Lisp's elegant design and contributes to its expressiveness and flexibility.
+
+And since we're in software engineering, let your curious mind enjoy something new and hopefully
+rewarding, or at least look at how many smileys you can put at the end of a function with all these
+closing parentheses!
```

Comment:
While I appreciate the sentiment of this, I fear that this comes across as a little preachy. In my personal opinion, I don't think we need to put down other languages in any capacity

Reply:
I agree, tried to make it less preachy now, yet showing some benefits to use it